guides/micronaut-openapi-generator-client/java/src/main/resources/application.yml

```diff
@@ -0,0 +1,30 @@
+base-path: "https://api.twitter.com/"
+context-path: "/"
+
+# tag::authorization[]
+micronaut:
+  application:
+    name: openapi-micronaut
+  server:
+    port: 8080
+  security:
+    oauth2:
+      clients:
+        twitter: # <1>
+          client-id: '${TWITTER_AUTH_CLIENT_ID}' # <4>
```

Comment:
add default values, so that a user can execute most tests even if he does not set the environment variables: 

```
client-id: '${TWITTER_AUTH_CLIENT_ID:xxx}' # <4>
client-secret: '${TWITTER_AUTH_CLIENT_SECRET:yyy}'
```